blog/2024-01-30-FAQ.md

```diff
@@ -0,0 +1,136 @@
+
+In the case of 10101, you can close a position without having to close your channel. Your funds will then be displayed on your off-chain balance, allowing you to operate further trades at your convenience. At any time, you can withdraw your funds to your on-chain balance by closing your DLC Channel 
+
+### How is it possible to trade directly from my wallet, without a platform?
+
+This is made possible thanks to **Discreet Log Contracts** (DLCs).
+
+A DLC is a multisig transaction output funded by two parties, which can be unilaterally spent by either one according to a future event _external_ to the blockchain. One or more oracles are used as trusted, oblivious third parties, as they attest to the outcome of the event and determine how the funds will be split.
+
+Applied to trading using 10101, a DLC is set up with the user providing part of the collateral and their counterparty in the trade providing the rest. In the case of BTCUSD perpetual swaps, the event external to the blockchain is the price of Bitcoin. The oracles eventually attest to the price and, depending on that value, the funds in the DLC can be split in a particular way, with the user realizing their PNL.
+
+Off-chain DLCs can be seen as an additional layer built on top of Bitcoin.
+
+If you want to learn more, we produced a series of [blog posts](https://10101.finance/blog/dlc-to-lightning-part-1) on this topic, and another one on the specific case of [trading settlements](https://10101.finance/blog/trade-settlement).
+
+### How is the DLC channel funded?
+DLC channels are bilaterally funded.
+
+We, on our side, will always put the necessary margin to cover the quantity (number of contracts) of your trade with a x2 leverage.
+
+Examples:
+- If the **opening price is $40.000** and you open a **$2.000 position** with a **x4 leverage**, your margin is **0,0125BTC**, ours is **0,025BTC**. 
+**Total channel value : 0,037BTC**
+- If the **opening price is $40.000** and you open a **$2.000 position** with a **x1 leverage**, your margin is **0,05BTC**, ours is **0,025BTC**. 
+**Total channel value : 0,075BTC**
+If the **opening price is $40.000** and you open **a $2.000 position** with a **x2 leverage**, the channel will be perfectly balanced as both parties will have a **0,025BTC margin**, for a **total channel value of 0,05BTC**
+
+Conclusion : your margin in BTC will depend on the opening price, quantity, and on the leverage you choose. Ours will always remain relative to the opening price and quantity you have selected
+
+**As your first trade defines the configuration of your channel, it's important to take the above elements into account when assessing your payout expectations.**
```

Comment:
> I mean, if that guy opened a new trade, we funded the other side of the channel with the corresponding margin, right?

Yeah, at the moment we are going with 50% of what they put up as margin. But that should be configurable soon.

> If that is correct : isn't our side of the channel the maximum payout he can expect as long as the channel is open as is? (if no splicing, no liquidity lending).

I now see where you are going with this! I wouldn't say it's the maximum payout, because that also includes their own margin i.e. payout is `margin +/- pnl - fees`. You could say that our side of the channel (minus order-matching fees) is the maximum amount they can earn with their first trade. And that amount changes as order-matching fees are paid to the coordinator.

Overall I think you are not wrong with what you wrote above and your comment, but maybe we can be a bit more explicit.